lms/djangoapps/discussion_api/api.py

```diff
@@ -554,12 +558,15 @@ def update_comment(request, comment_id, update_data):
     editable_fields = _get_comment_editable_fields(cc_comment, context)
     _check_editable_fields(editable_fields, update_data)
     serializer = CommentSerializer(cc_comment, data=update_data, partial=True, context=context)
-    if not serializer.is_valid():
-        raise ValidationError(serializer.errors)
-    # Only save comment object if the comment is actually modified
-    if update_data:
+    actions_form = CommentActionsForm(update_data)
+    if not (serializer.is_valid() and actions_form.is_valid()):
+        raise ValidationError(dict(serializer.errors.items() + actions_form.errors.items()))
```

Comment:
No. Even if they did, this would still work, just providing the error from actions_form.